### PR TITLE
[IMP] purchase: call for tenders 17.0 updates

### PR DESCRIPTION
**Pull request by samueljlieber**
Opening PR on behalf of Kara (kaek).

### CODE REVIEW

**Comment by StraubCreative** [comment]
Hi @hojo-odoo this is ready for your peer review, ty!

File: content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst [hunk 1]
```diff
@@ -9,9 +9,9 @@ Alternative RfQs
 .. |RfQ| replace:: :abbr:`RfQ (Request for Quotation)`
 .. |RfQs| replace:: :abbr:`RfQs (Requests for Quotation)`
 
-Sometimes, companies might want to request offers from multiple vendors at the same time, by
-inviting those vendors to submit offers for similar goods or services all at once. This helps
-companies to select the cheapest, fastest vendors, depending on their specific business needs.
+Sometimes companies might want to invite vendors to submit offers for similar goods or services all
```
Comment (hojo-odoo) [inline content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst:12]:
```suggestion
Sometimes, companies might want to invite vendors to submit offers for similar goods or services all
```

File: content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst [hunk 2]
```diff
@@ -263,46 +206,35 @@ without having an effect on the other live orders, once those orders have been c
...
 This causes a pop-up window reading :guilabel:`What about the alternative Requests for Quotations?`
-to appear. From the pop-up window, two options are presented: :guilabel:`Cancel Alternatives` and
-:guilabel:`Keep Alternatives`.
+to appear.
+
+To view a detailed form of one of the |RfQs| listed, click the line item for that quotation. This
+opens an :guilabel:`Open: Alternative POs` pop-up window, from which all details of that particular
+|RfQ| can be viewed.
+
+Once ready, click Close to close the pop-up window.
```
Comment (hojo-odoo) [inline content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst:219]:
```suggestion
Once ready, click :guilabel:`Close` to close the pop-up window.
```

**Review by hojo-odoo — APPROVED** [review]
Hey @kekiss,

Great job on this doc update! I just had a few small comments.

Thanks,
John

**Comment by kekiss** [comment]
@odoo/us-doc-review Ready for review!

**Review by ksc-odoo — APPROVED** [review]
hey @samueljlieber -- just finished my Final Review of this PR. Looks great! Just a few minor comments that require your attention. Once you implement those, feel free to move this to the next stage. Thanks! 👍

**Comment by kekiss** [comment]
Hi @samueljlieber !
This is ready for tech review!

**Review by samueljlieber — COMMENTED** [review]
Hey @kekiss! 
Nice work on this PR :) I have a couple quick suggestions for you, please see below!

Please remove the `calls-for-tenders-compose-email.png` image from the `calls_for_tenders/` media folder, as it is now unused and removed from the RST in your changes.

I cant request changes this PR, but please tag me in a comment for one more quick look after you've addressed these. Let me know if you have any questions! Thank you for your work!

**Comment by kekiss** [comment]
Hi @samueljlieber ! Tagging you for one more quick look!

**Comment by samueljlieber** [comment]
Hey @jero-odoo! Please tech review this PR, let me know if you have any questions! Thank you :)

**Review by jero-odoo — APPROVED** [review]
Hey @kekiss , this looks good! Thank you for your work on this one.
The one thing I wanted to call out was the icon on line 148 to the new formatting? (I know it wasn't included on the edits you made, but I think it is worth making the change now if you can).

`:icon:`fa-chevron-down` :guilabel:`(chevron)` icon`

Thanks so much!
...
Tagging @samueljlieber so he can delgate

**Comment by samueljlieber** [comment]
Thank you @jero-odoo! Looks good @kekiss, please consider updating the icon from Jess's review before merge. Thank you!
@robodoo delegate=kekiss

**Comment by kekiss** [comment]
Merging per delegation by @samueljlieber !

@robodoo r+

**Pull request closed without merging**